logic.py

```diff
@@ -185,6 +185,7 @@ def add_proposal(data):
             "category",
             "what_will_attendees_learn",
             "speaker_and_talk_history",
+            "talk_length",
```

Comment:
I added the talk_length to the pybay api here: https://github.com/pybay/pybay/pull/254
This was the change you helped me deploy to pybay staging the other night, and since then I deployed it to prod.

Reply:
will this create some integrity error with previous talks that we voted, that do not have this field?